backend/src/main/java/eventlistener/controller/EventController.java

```diff
@@ -0,0 +1,28 @@
+package eventlistener.controller;
+
+import eventlistener.model.event.Event;
+import eventlistener.service.EventService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/event")
+public class EventController {
+
+    private final EventService eventService;
+
+    public EventController(EventService eventService) {
+        this.eventService = eventService;
+    }
+
+    @GetMapping
+    public List<Event> getAllEvents(){
+        return eventService.getAllEvents();
+    }
+
+    @PostMapping
```

Comment:
wouldn't the proper 'rest' way be to add the id as a parameter here?

Reply:
afais there is no id yet as it will be created within eventService